patroni/dcs/kubernetes.py

```diff
@@ -576,6 +582,6 @@ def watch(self, leader_index, timeout):
             return True
 
         try:
-            return super(Kubernetes, self).watch(None, timeout)
+            return super(Kubernetes, self).watch(None, timeout + 0.5)
```

Comment:
the 0.5 looks a bit random, but ok

Reply:
It is [consistently](https://github.com/zalando/patroni/blob/master/patroni/dcs/etcd.py#L661) "random" :)

Reply:
then maybe create a constant or config for this offset :)